packages/react-form-renderer/src/files/condition.d.ts

```diff
@@ -28,6 +33,7 @@ export interface ConditionProps {
   values: object;
   children: React.ReactChildren;
   condition?: ConditionDefinition | ConditionDefinition[];
+  field: Field;
```

Comment:
Why was this added? I see that you have swapped it with `formOptions`. Did we forget to include for options inside this typing?

Reply:
`formOptions` were not used at all. the `field` is a object passed to `when` function.

Reply:
:+1: